### PR TITLE
Fix conda-store artifact/storage endpoints

### PR DESCRIPTION
**Pull request by viniciusdc**
## Reference Issues or PRs

#158

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: roles/traefik/templates/traefik_dynamic.yaml [hunk 1]
```diff
@@ -87,7 +87,7 @@ http:
       service: "dummy"
 
     minio:
-      rule: "Host(`{{ traefik_domain | default(hostvars[groups['hpc_master'][0]].ansible_ssh_host) }}`) && PathPrefix(`/minio`)"
+      rule: "Host(`{{ traefik_domain | default(hostvars[groups['hpc_master'][0]].ansible_ssh_host) }}`)"
```
Comment (viniciusdc) [inline roles/traefik/templates/traefik_dynamic.yaml:90]: Keeping the old `/mino` would require some changes on how conda-store prepares the URLs which would not be feasible at the moment

Comment (viniciusdc) [reply roles/traefik/templates/traefik_dynamic.yaml:90]: This looks like to be part of the S3 compatibility scheme, though couldn't find a ref. link besides this https://github.com/minio/minio-py/issues/1140#issuecomment-892683029 comment.

File: roles/traefik/defaults/main.yml [hunk 2]
```diff
@@ -34,4 +34,4 @@ conda_store_port: "5000"
 
 # role: minio
 minio_internal_port: "9001"
-minio_external_port: "9000"
+minio_external_port: "9080"
```
Comment (aktech) [inline roles/traefik/defaults/main.yml:37]: It's still pointing to `9080`? It would not work out of the box with App proxy.

**Comment by viniciusdc** [comment]
While this proves useful as is, the main issue with the redirection when behind a two-layer proxy, for example, is how conda-store computes the minio links behind the scenes. This is also present in the upstream cloud version of nebari. 

So to correctly address the issue of not exposing the port, we would need to refactor the signed links on conda-store first (as it is hardcoded to depend on the port and uses the endpoint as URL validation as well )

I think the best would be to close this issue, and first address this on upstream nebari, then apply the changes here. (in a new PR)

**Comment by aktech** [comment]
Closing it for now, as suggested by @viniciusdc

**Pull request closed without merging**